rpcs3/Gui/GameViewer.cpp

```diff
@@ -233,8 +233,10 @@ void GameViewer::RemoveGame(wxCommandEvent& event)
 {
 	long i = GetFirstSelected();
 	if (i < 0) return;
-	
-	fs::remove_all(vfs::get(m_path) + this->GetItemText(i, 6).ToStdString());
+
+	wxMessageDialog *dial = new wxMessageDialog(NULL, wxT("Permanently delete game files?"), wxT("Confirm Delete"), wxYES_NO | wxNO_DEFAULT);
```

Comment:
That correct for frames, not dialogs.
Look at examples, e.g. http://docs.wxwidgets.org/trunk/overview_cmndlg.html